### PR TITLE
moved my Component Library to new repo

### PR DESCRIPTION
**Pull request by knackofabhinav**

### CODE REVIEW

**Comment by rospyctrl** [comment]
This is looking all good abhinav.

**Comment by Gackrey** [comment]
I would suggest that to create a variable with global scope, declare it inside the :root selector instead of * selector, rest everything looks great to me.

**Comment by reetsrivastava** [comment]
Looks good to me Abhinav

**Review by SarthakSam — APPROVED** [review]
Good to go

**Review by k-yash — APPROVED** [review]
now everything looks great to me, Great work.

File: components.css [hunk 1]
```diff
@@ -0,0 +1,617 @@
...
+.alert{
+    border: 2px solid ;
+    display: flex;
+    align-items: center;
+    height: 2em;
+    margin: 2em;
+}
+
+.alert > i {
+    margin: 1em;
```
Comment (akshay-rajput) [inline components.css:40]: Use rem instead of em units

**Pull request merged**